Wireframe/index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
+    <title>Wire-frame</title>
     <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+</head>
+<body>
+    <!-- Header section -->
+    <header>
+        <div class="hero-image">
+            <img src="../Git img/githubimg.jpg" alt="Hero Image"/>
```

Comment:
It would typically make sense to nest any images under the `Wireframe` directory since that's where we're using them! For instance, you could create a directory similar to `css` (e.g. "img")

Reply:
Checked.